runtime/lua/vim/_editor.lua

```diff
@@ -886,9 +886,8 @@ end
 --- "Pretty prints" the given arguments and returns them unmodified.
 ---
 --- Example:
----
```

Comment:
Why remove this line

Reply:
No reason, I did it by mistake. I'll re-add it.